warrior/Actions/SeleniumActions/browser_actions.py

```diff
@@ -179,10 +200,23 @@ def browser_launch(self, system_name, browser_name="all", type="firefox",
                 browser_details = selenium_Utils.\
                     get_browser_details(browser, datafile=self.datafile, **arguments)
             if browser_details is not None:
+                if str(headless_mode).strip().lower() == "yes":
+                    try:
+                        from pyvirtualdisplay import Display
+                        display = Display(visible=0, size=(1024, 768))
+                        display.start()
+                        pNote("Running in headless mode")
+                    except ImportError:
+                        print_error("pyvirtualdisplay is not installed in order "
+                                    "to launch the browser in headless mode")
+                    except:
+                        print_error("Xvfb is not installed in order "
+                                    "to launch the browser in headless mode")
```

Comment:
Hi @Annam16,

If any of the exceptions mentioned above occur - the step should fail. Today, if I do not have pyvirtualdisplay/Xvfb installed on my system, it prints outs this exception, but eventually my step passes (because I do not really need the headless mode on my ubuntu); but the step should fail indicating that something went wrong.